Jenkinsfile

```diff
@@ -0,0 +1,41 @@
+@Library('pipeline-library') _
+
+webappPipeline {
+    slaveLabel = 'dev_v2'
+    nodeVersion = '10.16.2'
```

Comment:
currently, node 14 is the active lts. node 10 is about to hit EOL so we should see if we can use 14.

Reply:
Good point. I switched to to node 14